welcome.js

```diff
@@ -28,7 +28,18 @@ define(function(require, exports, module) {
         
         /***** Initialization *****/
         
-        var handle = editors.register("welcome", "URL Viewer", Welcome, []);
+        var targetWelcome = Welcome;
+        (function() {
+            var replacement = options.replacement,
+                pkg;
+            if (replacement) {
+                pkg = pkg || require(replacement.packagePath);
```

Comment:
If you need to replace welcome plugin with something else, it's better to do that in the config, by removing welcome and adding a new plugin, like https://github.com/c9/core/blob/master/configs/client-default-local.js#L154